CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 1.14.2 - 2024-04-26
```

Comment:
Missing a line after the date.

Reply:
```suggestion
## 1.14.2 - 2024-04-26

```

Reply:
You have just witnessed a master in laziness 🙇

Reply:
omg

Reply:
also ftr that newline doesn't actually impact the rendered markdown but it is good to be consistent